### DISCUSSION TITLE
Creator Project Proposal - Profile Section

### DISCUSSION THREAD

**Discussion by 0xNordian** [deso-protocol/dips#28] [Medium CIPs]
title: Creator Project Proposal
about: Creation of a profile section for creators promotion
labels: Project Proposal, Profile, Creators Promotion
assignees: N/A

**One-Line Description**
-------------------------
Creators must have a specific section in their profile to promote their projects to potential investors. The profile description is not enough.

**Explanation and Motivation**
--------------------------------
Theoretically, a creator coin is backed up by an idea, concept, content or project. In any case, a creator needs a proper way to explain the value proposition to the community. Therefore, I would like to propose creating a dedicated section in the profile to explain/promote a creator project and value proposition.

As a conceptual reference, you can check this link:
https://images.bitclout.com/f2b0ed676c9a3d151d899aa20a89f1b4f0b54ca98debebfc8053ea314834fdbe.webp

The sections and features of this section should be discussed and agreed upon; the image is just a reference to explain the concept. But video content, google drive embedding options or even native applications like [@pitchwall](https://bitclout.com/u/pitchwall?feedTab=Following) could fit perfectly in this feature.

**Some benefits:**
-------------------

1. Investors will have inside the platform a section to understand a creators project 
2. This content will facilitate the analysis for an investment (is not meant to substitute other methods will work as an important complement)
3. Creators will have a dedicated section in their profile to explain their projects, not needing to create content outside the platform with third-party apps.

**Comment by tijno** [comment]
screen linked to above embedded

![screen example](https://images.bitclout.com/f2b0ed676c9a3d151d899aa20a89f1b4f0b54ca98debebfc8053ea314834fdbe.webp)

I like this! 

Probably should be a more generic "About" section rather then Proposal. 

Not sure about technical implications of storing this on chain - but i guess its no more then 4 or 5 posts. 

Pinned post (already in progress) may be a decent temp workaround.

It could even offer a possible solution ?

Eg each of the sections in that tab are a pinned post:

post 1: main hero text
post 2: secondary section
post 3-5: portfolio posts (eg image or embeded video posts)

**Reply by dgsus** [reply]
awesome @N0rdian , @tijno i see this as just being a frontend visual implementation of pinned posts indeed

**Reply by lazynina** [reply]
Pinned posts, as currently being implemented, will allow users to pin multiple posts. Nodes will configure how many pinned posts they'd like to show at the top of a user's posts with a pinned icon.

**Reply by 0xNordian** [reply]
I agree that as a temporary solution, the pinned post will do. However, I think in the future, we will need something more 'formal' to present/promote the value proposition of our projects accordingly. Of course, this could be developed independently by node operators, but my suggestion is to use a solution like this in a standard way due to the platform's ethos, at least how I understand it.

Thanks for the feedback!

**Comment by HPaulson** [comment]
This is an amazing idea!! I think it'd be SUPERRRRRR nice if these had rich functionality, such as markdown. Github has had huge success with such, and they look amazing when utilized. However, no idea how this would be handled on-chain, and would likely be unviable as such -- just tossing it out there if someone somehow has an idea of how this could be possible.

I agree with Tijn re: calling it "About", and think that using the Pin feature would be a solution to such -- one small change would have to be altered to allow multiple pins. This would be a massive benefit though, as the bio simply isn't enough to describe yourself. It'd also be nice if this section contained connections (Such as github/twitter badge, etc, see discord as an example of usage) or other fields like this.

**Reply by dgsus** [reply]
yes multiple! but perhaps as i mentioned above just a frontend visual implementation of it?

**Reply by AikonMishima** [reply]
Agreed as this would be useful for the bitclout.com clones in the future, yet less so for node runners with different applications/executions.

**Comment by dylsell** [comment]
Love this proposal, recently recorded some thoughts of my own on this https://vimeo.com/manage/videos/570408722 (start video at 4m) 

See also 😉
<img width="703" alt="Screen Shot 2021-07-09 at 5 01 00 PM" src="https://user-images.githubusercontent.com/33188041/125135713-77f3f800-e0d7-11eb-9bf2-bf6d58fbb76f.png">

**Comment by AikonMishima** [comment]
Big up-vote from my end as my current workaround for showcasing my projects was a link tree alternative called « flow page ».

An effective introduction of other links in the tab as presented in the image would also be great. Examples include:
- Other socials
- Methods of communication (telegram/discord)
- Redirects to current projects
- White paper of the project (if the creator coin is a project)

Excellent idea, @N0rdian. You’re on fire!

**Comment by omicreativedev** [comment]
I feel as though pinned posts could work all alone. But then we broach the conversation about post length. This main node submit post form only supports limited characters. I've not seen any issue with plugins and other nodes using long posts. But presently Bitclout.com submit form doesn't support long posts.

Have we ever considered adding that to settings? Such as a on/off long posts? (off would make long posts visible but trunicated with 'read more' and not postable — on would make long posts postable and fully readable without read more link) 

I don't see why we couldn't support basic markdown. It might look weird in other nodes but I imagine other dapps will one day have some pretty strange stuff in posts (maybe even code itself) so I imagine it would eventually be up to each node how to interpret or block/filter various "stuff" posted to the blockchain from other post interfaces.

Alternatively, there is pseudo-formatting, like bold, could be done like 𝐅𝐀𝐊𝐄 𝐁𝐎𝐋𝐃 𝐓𝐄𝐗𝐓 using a text style generator (like carsen's emoji modal dropdown but add text formatting tab) see example: https://maketext.io/fancy/ (not this one obviously, maybe an open source one?) Anyhow, it's "mock" bold, italic, script, etc. has rendered as well as emoji's on modern browsers and smartphones.

Posts already support embed AND image in the same post and cloutfeed supports embed and multiple images.  A pinned post on cloutfeed could look like this with a video, multiple images, fake formatting, emoji, etc. without anything really extra added. 

Once pinned posts are fully deployed, I'm sure people will find creative uses like this.

I DO think all profiles should support social links. But again, a linktree or shortlink of choice is just as well.

Lastly, I think the Bio can actually support more characters. We could just triple the bio size and add little arrow to view it. Wouldn't solve for media, but again- pinned posts...
